app/src/main/kotlin/com/simplemobiletools/flashlight/helpers/MyCameraImpl.kt

```diff
@@ -55,6 +48,9 @@ class MyCameraImpl(val context: Context) {
     }
 
     fun toggleStroboscope(): Boolean {
+        handleCameraSetup()
```

Comment:
Do we want maximum brightness when we are in stroboscope mode?

Reply:
lets always user defined brightness level, that he set at the flashlight

Reply:
This is the current behaviour